automation/README.md

```diff
@@ -0,0 +1,77 @@
+# Automation with oVirt CI
+
+In order to keep playbooks in this repository operational,
+it is being integrated with [oVirt CI System][ovirt-ci-system-doc].
+Everything located under `automation` directory is related to integration
+to [oVirt CI System][ovirt-ci-system-doc].
+
+There is a list of events which can trigger a job on [oVirt CI System][ovirt-ci-system-doc],
+The KubeVirt Ansible repository uses the `check-patch` event at the moment,
+to verify incoming patches.
+
+Once you submit a PR for this repository, the [oVirt CI System][ovirt-ci-system-doc]
+triggers [Jenkins job](http://jenkins.ovirt.org/blue/organizations/jenkins/kubevirt_kubevirt-ansible_standard-check-pr/activity)
+which executes the [`automation/check-patch.sh`](./check-patch.sh) script.
+
+This script will execute the Ansible playbook [`./control.yml`](../control.yml)
+which wraps entire testing flow.
+
+
+## `control.yml` Ansible playbook
+
+This Ansible playbook is wrapper for entire testing flow which is composed from
+following steps (partial playbooks)
+* [provision testing environment](#testing-environment)
+* [deploy OpenShift](#deploy-openshift)
+* [install KubeVirt](#install-kubevirt)
+
+This playbook is executed inside of CentOS 7.4 mock.
+There is additional software installed, please read
+[automation/check-patch.packages](./check-patch.packages) for complete list.
+
+Parameters and usage of this playbook is described at
+[Deploy new Kubernetes or OpenShift cluster and KubeVirt with Lago](../README.md#deploy-new-kubernetes-or-openshift-cluster-and-kubevirt-with-lago).
+
+This is list of playbooks which are executed.
+* [control.yml](../control.yml)
```

Comment:
Let's wait until we finish with the re-org and then merge this doc (the playbooks names are going to be changed).

Reply:
OK, I will keep it updated, but I would like to get it in before or on Wed.